package.json

```diff
@@ -58,8 +66,11 @@
     "type-detect": "^4.0.8"
   },
   "devDependencies": {
+    "@babel/core": "^7.1.6",
```

Comment:
Ugh! How long do you think it'll be before people start suggesting that we rewrite the entire codebase in stage-0 ES?

Reply:
We've been fighting it successfully so far 😉

Reply:
"But you're already loading babel..."

Reply:
It's actually not such a bad idea to verify that sinon works fine when babelified. However, the one strong argument that we can always pull is that Sinon works very low level and we need complete control over the code that runs. Therefore, the codebase is written soley with language features that are supported by the targeted runtimes natively.

Reply:
If it becomes a problem to hold off the stage-0 hordes, I'll take a hammer to the problem